### PR TITLE
Update interpretation and comment views to dashboard design (master)

### PR DESCRIPTION
**Pull request by tokland**
Closes https://jira.dhis2.org/browse/DHIS2-5469

* `dateUtils.formatDate()` logic taken from dashboard-app.

screenshot from 2019-01-14 10-51-14

### CODE REVIEW

File: src/ui/EastRegion.js [hunk 1]
```diff
@@ -542,17 +543,22 @@ EastRegion = function(c) {
                                 }
                             }, {
                                 xtype: 'label',
-                                text: comment.text,
+                                style: 'color: #666; float: right',
+                                hidden: !interpretation.access || !interpretation.access.update,
```
Comment (edoardo) [inline src/ui/EastRegion.js]: This prevents the comment date to be shown, but the comment text is shown regardless of the access rights.

Comment (tokland) [reply src/ui/EastRegion.js]: That was old code, but it makes no sense as it is. Date and text should be visible even if the user cannot update a comment. So I've removed the hidden tag, does it make sense to you? (I'll update the other PR also, when confirmed)

Comment (edoardo) [reply src/ui/EastRegion.js]: I think it should be fine. The access control should be on the links. If a comment is visible, the whole comment box including the date should be visible.

Comment (tokland) [reply src/ui/EastRegion.js]: Cool, commit also applied to the v30 branch.

**Pull request merged**